tests/core/job/test_job.py

```diff
@@ -80,17 +81,17 @@ def _error():
     raise RuntimeError("Something bad has happened")
 
 
-def test_create_job(pipeline, task, job):
-    from src.taipy.core.pipeline._pipeline_manager_factory import _PipelineManagerFactory
+def test_create_job(scenario, task, job) -> None:
```

Comment:
```suggestion
def test_create_job(scenario, task, job):
```
This is SUS. Copilot? 🤣

Reply:
hahaha thanks! I think it's the Sonarlint suggestion extension :)